app/models/post.rb

```diff
@@ -1,3 +1,21 @@
 class Post < ApplicationRecord
   belongs_to :user
```

Comment:
- [ ] **Optional:** Consider renaming `user` to `author` to match the EMD diagram.